### ISSUE TITLE
Update peerDependencies requirements to support TS 3

### ISSUE THREAD

**Issue by artola** [angelozerr/tslint-language-service#77]
The package requires `"typescript": ">= 2.3.1 < 3"`, then this warning is printed when running together to TS 3.

```
npm WARN tslint-language-service@0.9.9 requires a peer of typescript@>= 2.3.1 < 3 but none is installed. You must install peer dependencies yourself.
```

Would be nice to update the `peerDependencies` to support TS 3, I did not find any problem while using it, but might be recommended check it with the tests.

**Comment by angelozerr** [comment]
Any PR are welcome!

**Comment by artola** [comment]
[PR](https://github.com/angelozerr/tslint-language-service/pull/78) is not passing CI.
I do not understand why when the change is just "peerDependencies" and should impact only in the user's environment.
Any hint?

**Comment by juansalvatella** [comment]
Has anyone been able to check this out?

**Comment by artola** [comment]
> Has anyone been able to check this out?

Might be that you are interested to replace this plugin with the newest: **typescript-tslint-plugin**.
